rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/TaskResultImpl.java

```diff
@@ -119,6 +121,11 @@ public boolean isRaw() {
         return isRaw;
     }
 
+    @Override
+    public Map<String, Serializable> getResultMap() {
```

Comment:
as this is not handled (it will always return null), I think it would be better to return a UnsupportedOperationException or to handle it (but it will take some time : db, rest mappings)